docs/core/tools/csproj.md

```diff
@@ -69,6 +69,13 @@ This change does not modify the main mechanics of other includes. However, if yo
 ### Recommendation
 With csproj, we recommend that you remove the default globs from your project and only add file paths with globs for those artifacts that your app/library needs for various scenarios (runtime, NuGet packaging, etc.)
```

Comment:
That is not part of this pull request. I'm fine fixing it here, but it doesn't sound right.